src/_blog/go-libp2p-2022.md

```diff
@@ -0,0 +1,118 @@
+---
+tags:
+  - libp2p
+  - update
+  - Go
+title: go-libp2p in 2022
+description:
+date: 2023-01-11
+permalink: ''
+translationKey: ''
+header_image: /
+author: Prithvi Shahi
+---
+
+We are excited to share with you all the progress that has been made on [go-libp2p](https://github.com/libp2p/go-libp2p) in 2022. It has been a year full of exciting new features, code organization, and a growing team of talented contributors.
+
+Throughout the year, we released seven updates to go-libp2p ranging from [v0.18.0](https://github.com/libp2p/go-libp2p/releases/tag/v0.18.0) to [v0.24.0](https://github.com/libp2p/go-libp2p/releases/tag/v0.24.0), with a number of patch releases in between. We also welcomed [Marco](https://github.com/MarcoPolo) to the go-libp2p engineering team, bringing the total number of team members to two (alongside [Marten](https://github.com/marten-seemann))😀. In total, we had [21 contributors](https://github.com/libp2p/go-libp2p/graphs/contributors?from=2022-01-01&to=2022-12-31&type=c) to the project in 2022.
+
+Without further ado, let's take a look at notable accomplishments in the last year:
+
+## New Features ✨
+
+### Transport Protocols 🚚
+
+#### WebTransport 📡
+
+One of the most exciting developments of the year was the release of the WebTransport protocol in [v0.23.0](https://github.com/libp2p/go-libp2p/releases/tag/v0.23.0). WebTransport enables browser-to-server connectivity in go-libp2p when paired with a peer running [js-libp2p-webtransport](https://www.npmjs.com/package/@libp2p/webtransport) in the browser.
+
+To learn more about this exciting feature, check out our blog post on [WebTransport in libp2p](https://blog.libp2p.io/2022-12-19-libp2p-webtransport/) and the [WebTransport documentation](https://docs.libp2p.io/concepts/transports/webtransport/).
+
+#### WebRTC (Browser to Server) 🛰️
+
+In addition to WebTransport, the go-libp2p team also began work on enabling the WebRTC transport, in partnership with Little Bear Labs. This new transport will allow for connectivity between go-libp2p server nodes and js-libp2p browser nodes using [js-libp2p-webrtc](https://github.com/libp2p/js-libp2p-webrtc). 
+
+While this feature is still under development and testing, you can check out a proof-of-concept demo of it in action on our [libp2p Day 2022 Recap blog post](https://blog.libp2p.io/2022-11-22-libp2p-day-2022-recap/#why-webrtc). 
+
+We expect to release this feature in Q1 2023.
+
+
+#### QUIC 🐰
+
+We also made developments to the existing QUIC implementation in go-libp2p. While go-libp2p has supported QUIC for some time, we upgraded to the latest version, [RFC 9000](https://datatracker.ietf.org/doc/), from the [draft-29 version](https://datatracker.ietf.org/doc/html/draft-ietf-quic-transport-29) (in [v0.24.0](https://github.com/libp2p/go-libp2p/releases/tag/v0.24.0)), inspired by the [alpha release of the QUIC implementation in rust-libp2p](https://github.com/libp2p/rust-libp2p/releases/tag/v0.50.0).
```

Comment:
This is not entirely correct, go-libp2p nodes have supported RFC 9000 for quite a while. It's just that we'd _dial_ draft-29 by default.

The only thing that we really changed was that we're now properly distinguishing between the two versions in their multiaddress.

Reply:
Updated it a bit.